### DISCUSSION TITLE
Update Aurelia Plugin

### DISCUSSION THREAD

**Discussion by CollinHerber** [storybookjs/storybook#24548] [Ideas]
### Is your feature request related to a problem? Please describe.

I would love to see the following updated:
https://github.com/storybookjs/storybook/tree/master/app/aurelia

Aurelia is such an incredible framework and supporting it would be great.

Aurelia 2 support would be ideal, while it's currently in Beta it is very stable and am using it in several production applications.

### Describe the solution you'd like

Supporting Aurelia 1 and Aurelia 2
I could see just Aurelia 2 support being fine.

### Describe alternatives you've considered

There is this: https://github.com/ekzo-dev/storybook-aurelia

But it does not appear to be updated for Storybook 7 which is a must for my project.

### Are you able to assist to bring the feature to reality?

yes, I can

### Additional context

Happy to work with the team, or even contribute code/knowledge for this effort.

Thank you Storybook for all you've done for me.

**Comment by valentinpalkovic** [comment]
Hi @CollinHerber,

Thank you for being so interested in supporting an Aurelia 1 and 2 integration.

Usually, new frameworks or frameworks that are not widely spread are not maintained by the core team. Feel free to support the [mentioned project](https://github.com/ekzo-dev/storybook-aurelia) to make it ready for Storybook 7 and even to support Aurelia 2. If the Aurelia Framework integration for Storybook becomes popular and is widely used, we can consider making it part of the mono-repository to be maintained by the core maintainers.

Also, feel free to ask us on Discord at any time if there are questions popping up regarding the integration. We try to do our best to answer them.

I hope it is okay for you if I close this discussion since I believe that the discussion/issue should be recreated in https://github.com/ekzo-dev/storybook-aurelia. The [linked](https://github.com/storybookjs/storybook/tree/master/app/aurelia) Aurelia folder is also not part of the mono repository anymore. Our main development branches are `next` and `main`. `master` is stale and is not in use anymore.

Let me know if you still have open questions!

**Reply by CollinHerber** [reply]
In storybook 6 you supported it as apart of the the cli project. Are you saying the core team is no longer going to support it? Neither version

**Reply by valentinpalkovic** [reply]
[That's correct](https://storybook.js.org/docs/7.4/react/configure/frameworks#legacy-framework-support). In Storybook 7.0, we have removed the core support for [Aurelia](https://github.com/aurelia/framework), [Marionette](https://github.com/marionettejs/backbone.marionette), [Mithril](https://github.com/MithrilJS/mithril.js), [Rax](https://github.com/alibaba/rax), and [Riot](https://github.com/riot/riot)

**Reply by CollinHerber** [reply]
Olay thank you for the clarification.
